sbc_compassion/wizards/import_letters_test.py

```diff
@@ -0,0 +1,107 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Loic Hausammann <loic_hausammann@hotmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+import base64
+
+from openerp import models, fields, _, api
+from ..tools import import_letter_functions as func
+
+
+class TestImportLetters(models.TransientModel):
+    """ Class testing the letters import by highlighting the keypoints of
+    the import (image used, keypoints found for the templates, showing only
+    the line not well detected, ...)
+    """
+    _inherit = 'import.letters.history'
+    _name = 'test.import.letters.history'
+    _description = _("""Easier way to test the import""")
+    csv_file = fields.Many2many(
```

Comment:
I forgot to answer to the question. Yes it can (it is easy to deal with it in the code and the user will not have to create a new csv every time he wants to mix some test files together)